Misc/NEWS.d/next/Security/2020-06-21-19-01-01.bpo-41004.P6i7Nj.rst

```diff
@@ -0,0 +1,2 @@
+The hash() methods of classes IPv4Interface and IPv6Interface (of the ipaddress library) had issue of generating constant hash values of 32 and 128 respectively causing hash collisions.

```

Comment:
This should be `__hash__()`, not `hash()`. 

I think the whole sentence would read better as:
```
The __hash__() methods of  ipaddress.IPv4Interface and ipaddress.IPv6Interface incorrectly 
generated constant hash values of 32 and 128 respectively. This resulted in always causing 
hash collisions.
```